ios/posthog_flutter.podspec

```diff
@@ -21,8 +21,8 @@ Postog flutter plugin
   s.ios.dependency 'Flutter'
   s.osx.dependency 'FlutterMacOS'
 
-  # ~> Version 3.30.1 up to, but not including, 4.0.0
-  s.dependency 'PostHog', '>= 3.30.1', '< 4.0.0'
+  # ~> Version 3.31.0 up to, but not including, 4.0.0
+  s.dependency 'PostHog', '>= 3.31.0', '< 4.0.0'
```

Comment:
https://github.com/PostHog/posthog-flutter/blob/42c130df874f1bd0fb3134b119b36a287909dd4a/ios/Classes/PosthogFlutterPlugin.swift#L143
we should default to `false` then until its better? otherwise if something happens, the ios bits will be enabled still